app/models/audio_file.rb

```diff
@@ -48,4 +35,24 @@ class AudioFile < BaseModel
   def fixerable_final?
     [VALID, COMPLETE, TRANSFORMING, TRANSFORM_FAILED, TRANSFORMED].include? status
   end
+
+  def compliant_with_template?
+    status_msg.nil?
+  end
+
+  def validate_on_template
```

Comment:
I've nuked case #4 now, so INVALID will only mean invalid by template. if the lambda has determined the file to be invalid, it'll be called FAILED.